coc/events.py

```diff
@@ -765,6 +781,34 @@ def _task_callback_check(self, result):
             self._updater_tasks[name] = self.loop.create_task(lookup[name]())
             self._updater_tasks[name].add_done_callback(self._task_callback_check)
 
+    async def _raid_poller(self):
+        # pylint: disable=broad-except, protected-access
+        try:
+            age = 0
+            while self.loop.is_running():
+                try:
+                    [raidlog_entry] = await self.get_raidlog("#2PP", limit=1)
+                    raidlog_entry: coc.raid.RaidLogEntry
+                except Maintenance:
+                    await asyncio.sleep(15)
```

Comment:
Magic integer should be a constant with a comment above if the name is not descriptive.